src/components/pages/data/state-nav-no-script.js

```diff
@@ -14,7 +14,7 @@ export default ({ stateList }) => (
       <ul>
         {stateList.map(state => (
           <li key={state.node.state}>
-            <Link to={`/data#state-${state.node.state}`}>
+            <Link to={`/data#state-${state.node.state.toLowerCase()}`}>
```

Comment:
Let's change these to regular `a` elements with an href so that there's no page load when you click on them:

```jsx
<a href={`#state-${state.node.state.toLowerCase()}`}>
```

Reply:
done in 5da01e54d50386663e05ea506b88b0c6e8f58989 😄